docs/components/modal/README.md

```diff
@@ -71,19 +76,19 @@ via the `modal-footer` slot. Note when using the `modal-footer` slot the default
 slot the default header `X` close button will not be present, nor can you use
 the `modal-title` slot.
 
+## Toggle Modal Visibility
 
-### Toggle Modal Visibility
 There are several methods that you can employ to toggle he visibility of `<b-modal>`.
 
-#### Using `v-b-modal` directive (recommended)
+### Using `v-b-modal` directive (recommended)
 
```

Comment:
This should be 4 hashes, or change th ones below to 3 hashes, since the headings are at the same sub level under Toggle Modal Visibility